src/shared/components/ncTable/sections/CustomTable.vue

```diff
@@ -32,7 +32,7 @@
 				<tr />
 			</tbody>
 		</table>
-		<div class="pagination-footer" :class="{'large-width': !appNavCollapsed || isMobile}">
+		<div v-if="pageNumber > 1" class="pagination-footer" :class="{'large-width': !appNavCollapsed || isMobile}">
```

Comment:
```suggestion
		<div v-if="totalPages > 1" class="pagination-footer" :class="{'large-width': !appNavCollapsed || isMobile}">
```

Wouldn't we otherwise use the current page?

Reply:
🙈 sorry